### DISCUSSION TITLE
[Bug]: Issue changing Hybrid meeting to just in-person

### DISCUSSION THREAD

**Discussion by VenturaCntyCentralOffice** [code4recovery/12-step-meeting-list#614] [Q&A]
### Contact Details

vcaaco@verizon.net

### What happened?

If I have a meeting listed that is currently online and in-person and the meetings changes to in-person only I experience a problem.
After I remove the URL and the URL note and click on update, I get a pop-up box at the URL line that says "!Please enter a URL." I have tried this on Chrome and Firefox with the same result, however it does work on my android phone.   
 

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Firefox, Chrome

**Comment by brianw-area24** [comment]
Did you ever figure this one out? I've looked at it a couple times, and the only way I've been able to duplicate the issue is if I left a space character in the URL field. I was not able to duplicate it when the URL field was completely empty.

If you're still having issues, maybe post a screen shot of that part of the form would help us figure out what's going on.

Brian W.
